quickstarts/mlops/truera/config.yml

```diff
@@ -0,0 +1,51 @@
+id: d0e829a6-ede4-4933-9065-9f0c56f8aa7e # TODO(vivek): How to generate?
```

Comment:
Please remove this `id` as these will get auto-generated when the PR is merged
```suggestion
```

Reply:
I ran into a few issues while exporting fixes for comments, created new PR instead. Please review it.
https://github.com/newrelic/newrelic-quickstarts/pull/566